README.md

```diff
@@ -1,36 +1,39 @@
-# OOP school library
+# Catalog of my things
 
 <a name="readme-top"></a>
 
-# ￼ Table of Contents
+# Table of Contents
 
-- [￼ About the Project](#about-project)
-  - [￼ Built With](#built-with)
+- [About the Project](#about-project)
+  - [Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-- [￼ Getting Started](#getting-started)
+- [Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
   - [Deployment](#triangular_flag_on_post-deployment)
-- [￼ Authors](#authors)
-- [￼ Future Features](#future-features)
-- [￼ Contributing](#contributing)
-- [￼ Show your support](#support)
-- [￼ Acknowledgements](#acknowledgements)
-- [￼ FAQ](#faq)
-- [￼ License](#license)
+- [Authors](#authors)
+- [Future Features](#future-features)
+- [Contributing](#contributing)
+- [Show your support](#support)
+- [Acknowledgements](#acknowledgements)
+- [FAQ](#faq)
+- [License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# ￼ [OOP school library] <a name="about-project"></a>
+# [Catalog of my things] <a name="about-project"></a>
 
-**[OOP school library]** is like librarian of OOP for a University, and it is a tool to record what books are in the library and who borrows them. The app that I will create will allow you to:
-  
+**[Catalog of my things]** "Catalog of my things" is a simple console app that allows users to manage collections of the things they own. It has been developed based on the following UML class diagram.
 
-## ￼ Built With <a name="built-with"></a>
+[Video presentation](https://drive.google.com/file/d/1l9gVw0djCoo-PJS55vtxsyNlixtV4C3y/view?usp=drive_link)
```

Comment:
- [ ] Great job working on the console application. I could not view your video presentation because of acces rights. Kindly change the access rights of the video.
![image](https://github.com/BranBayou/catalog-of-my-things/assets/62849872/bd490850-dc79-4228-9275-048ff950eabe)